src/compute-engine/boxed-expression/rules.ts

```diff
@@ -419,7 +419,7 @@ function parseRule(
 
         // Check for conditions
         const conditions = parseModifierExpression(parser);
-        if (conditions === null) return null;
+        if (conditions === null) return `${prefix}${id}`;
```

Comment:
Believe the fix of this typo has resulted in correct parsing of LaTeX with sequence-syntax (`...n + ...`)